homeassistant/components/tado/climate.py

```diff
@@ -767,3 +800,20 @@ def _control_hvac(
             vertical_swing=vertical_swing,  # api defaults to not sending verticalSwing if swing not None
             horizontal_swing=horizontal_swing,  # api defaults to not sending horizontalSwing if swing not None
         )
+
+    def _does_current_hvac_mode_have_setting(self, setting: str) -> bool:
+        return (
+            self._current_tado_capabilities.get(self._current_tado_hvac_mode, {}).get(
+                setting
+            )
+            is not None
+        )
+
+    def _does_current_tado_setting_is_managed_by_current_hvac_mode(
```

Comment:
```suggestion
    def _is_current_setting_managed_by_current_hvac_mode(
```

Reply:
Maybe supported is a better name instead of managed, but depends on how its used